### PR TITLE
[DOC-31] JETI docs

### PR DESCRIPTION
**Pull request by awmuncy**
See diff or deployment for changes.

### CODE REVIEW

File: docs/sdk/plugins/JETI.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+### 1. Installation
+
+First, install the JETI package using npm or yarn:
+
+```
+npm i @tableland/jeti @tableland/sdk
+```
+
+### 2. Import the necessary modules from @tableland/sdk and @tableland/jeti:
```
Comment (dtbuchholz) [inline docs/sdk/plugins/JETI.md:20]: can probs just state `Import the necessary modules` and drop `from @tableland/sdk and @tableland/jeti:`

File: docs/sdk/plugins/JETI.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+Get the data you want to add to IPFS as a Blob object, and use the `prepare` template function to insert it into a query:
+
+```javascript
+const avatar = new Blob([
+  /* avatar img file contents here */
+]);
+
+const preparedQuery =
+  await prepare`INSERT INTO ${create.txn?.name} (name, avatar_cid) VALUES ('Murray', ${avatar});`;
```
Comment (dtbuchholz) [inline docs/sdk/plugins/JETI.md:54]: prepare needs `()` aka <code>prepare(`INSERT INTO ...`)</code> and same thing with the `?`

File: docs/sdk/plugins/JETI.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+const receipt = await db.query(preparedQuery);
```
Comment (dtbuchholz) [inline docs/sdk/plugins/JETI.md:56]: `query` isn't a function, so i'm assuming `db.prepare(...).run()`?

File: docs/sdk/plugins/JETI.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+const db = new Database();
+
+const { meta: create } = await db
+  .prepare(
+    `CREATE TABLE my_sdk_table (id integer primary key, name text, avatar_cid text);`
+  )
+  .run();
+
+// The table's `name` is in the format `{prefix}_{chainId}_{tableId}`
+console.log(create.txn?.name); // e.g., my_sdk_table_80001_311
```
[the remark this answers is not in the document]
Comment (awmuncy) [reply docs/sdk/plugins/JETI.md:41]: Maybe I'm missing something, but this should be standard JS syntax.

The `const { meta: create } = await db()` looks a little typescript-y, but all it's doing is assiging the property of `meta` to the `create` const.

[the remark this answers is not in the document]
Comment (dtbuchholz) [reply docs/sdk/plugins/JETI.md:41]: oh, no way lol...i always thought that was a TS-only thing. nice.

File: docs/sdk/plugins/JETI.md (hunk 2 above)
Comment (dtbuchholz) [reply docs/sdk/plugins/JETI.md:54]: nvm, template literal

**Comment by awmuncy** [comment]
Hey @dtbuchholz, this has changed substantially since you last reviewed (many thanks to your feedback!) Not just the docs, but the underlying library. Let me know what you think of the changes.

**Comment by dtbuchholz** [comment]
@awmuncy nice, sgtm! I can give it another go and see if anything comes up in my testing but looks solid.

**Comment by awmuncy** [comment]
Oh! You'll need to pull the branch off which this PR is built: https://github.com/tablelandnetwork/jeti/pull/55. The skip function isn't in the staging branch.

**Pull request merged**